ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -497,7 +497,7 @@
              [k v])))
 
 (def new-data-example [{:starting-week :foo
-                        :different-week :foo }])
+                        :different-week :foo}])
```

Comment:
Looks like un-used code. Could this be removed?

Reply:
Good catch. Will remove